<launch>
    <arg name="rname" default="laikago"/>
    <param name="robot_name" value="$(arg rname)"/>

    <!-- <node pkg="unitree_controller" type="unitree_servo" name="unitree_servo" respawn="true"/> -->

</launch>